code/datums/wires/_wires.dm

```diff
@@ -41,6 +41,9 @@
 	/// If every instance of these wires should be random. Prevents wires from showing up in station blueprints.
 	var/randomize = FALSE
 
+	/// The trait someone needs in order to see these wires
+	var/my_wire_trait = TRAIT_KNOW_ENGI_WIRES
```

Comment:
This should default to null. TRAIT_KNOW_ENGI_WIRES should be set for any wire that players with the relative skillchip should know about, so that we don't end up with contributors adding new wires datums that have nothing to do with the department being revealed to engineers.